src/ape/__init__.py

```diff
@@ -41,24 +41,33 @@
 
 # Main types we export for the user
 
-compilers = _CompilerManager(config, plugin_manager)  # type: ignore
+compilers = _CompilerManager(config=config, plugin_manager=plugin_manager)
 """Manages compilers for the current project. See
 :class:`ape.managers.compilers.CompilerManager`."""
 
-networks = _NetworkManager(config, plugin_manager)  # type: ignore
+networks = _NetworkManager(config=config, plugin_manager=plugin_manager)
 """Manages the networks for the current project. See
 :class:`ape.managers.networks.NetworkManager`."""
 
-_converters = _ConversionManager(config, plugin_manager, networks)  # type: ignore
+_converters = _ConversionManager(
+    config=config,
```

Comment:
Specifying param names makes it seem like these are optional

Reply:
I am guessing pydantic requires this or something because of the implicit constrors

Reply:
Pydantic requires this yeah. I think it throws if you fail to provide a parameter with no default in the struct class